teacher.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
+
+teacher = Teacher.new(30, 'Mathematics', 'Jack')
+puts teacher.can_use_services?
```

Comment:
- [ ] [OPTIONAL] Kindly remove the console logs in order to have a clean code base.